x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -0,0 +1,277 @@
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.useMachineLearningToLabel',
+              {
+                defaultMessage:
+                  'Use machine learning to automatically detect anomalies.',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+    {
+      id: 'infra',
+      label: i18n.translate(
+        'xpack.observability_onboarding.experimentalOnboardingFlow.euiCheckableCard.monitorMyInfrastructureLabel',
+        { defaultMessage: 'Monitor my infrastructure' }
+      ),
+      description: (
+        <ul>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.builtOnPowerfulElasticsearchLabel',
+              {
+                defaultMessage:
+                  'Built on powerful Elasticsearch, stream in and scale infrastructure metrics from your systems, cloud, network, and other infrastructure sources',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.getLogicalAndPhysicalLabel',
+              {
+                defaultMessage:
+                  'Get logical and physical views of your infrastructure topology.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.breakDownApplicationAndLabel',
+              {
+                defaultMessage:
+                  'Break down application and infrastructure silos for faster root cause detection',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+  ];
+
+  const [selectedId, setSelectedId] = useState<string>();
+  const [hoveredId, setHoveredId] = useState<string>();
+
+  const visibleOption = hoveredId
+    ? options.find((option) => option.id === hoveredId)
+    : selectedId
+    ? options.find((option) => option.id === selectedId)
+    : undefined;
+
+  return (
+    <EuiPanel hasBorder>
+      <TitleWithIcon
+        iconType="indexRollupApp"
+        title={i18n.translate(
+          'xpack.observability_onboarding.experimentalOnboardingFlow.strong.startCollectingYourDataLabel',
+          {
+            defaultMessage:
+              'Start collecting your data by selecting one of the following use cases',
+          }
+        )}
+      />
+      <EuiSpacer size="m" />
+      <EuiFlexGroup css={{ margin: 20 }}>
+        <EuiFlexItem css={{ fontWeight: 'bold' }}>
+          {options.map((option, index) => (
+            <div
+              key={option.id}
+              onMouseEnter={() => setHoveredId(option.id)}
+              onMouseLeave={() => setHoveredId(undefined)}
+            >
+              {index > 0 && <EuiSpacer size="m" />}
```

Comment:
Could this be handled by a flex group and flex item per option (the gutter size would define the sapce between the items)?

Reply:
The layout could be achieved using flex group but that would cause flickering with the hover effect. Using a spacer ensures that the spacing is part of the hit area

Reply:
Makes sense, could you add a comment to the code to explain?